### PR TITLE
Update to version 1.5.3

### PR DESCRIPTION
**Pull request by shamsimam**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,9 +4,13 @@ All notable changes to this project will be documented in this file
  
 The format is based on [Keep a Changelog](http://keepachangelog.com/) and this project adheres to [Semantic Versioning](http://semver.org/).
 
-## [Unreleased]
+## [1.5.3] - 2017-07-17
 ### Added
+- The Framework ID is now configurable, from @dposada
```
Comment (dposada) [inline CHANGELOG.md]: Maybe "Mesos Framework ID"?

**Comment by shamsimam** [comment]
Please use **rebase-and-merge option** while merging this PR.

File: scheduler/project.clj [hunk 2]
```diff
@@ -13,7 +13,7 @@
 ;; See the License for the specific language governing permissions and
 ;; limitations under the License.
 ;;
-(defproject cook "1.5.3-SNAPSHOT"
+(defproject cook "1.5.3"
```
Comment (dposada) [inline scheduler/project.clj]: I would favor having this move to `1.5.4-SNAPSHOT`, with a commit containing `1.5.3` that we can tag for the release (and then not squash the commits)

File: CHANGELOG.md [hunk 3]
```diff
@@ -4,9 +4,15 @@ All notable changes to this project will be documented in this file
...
 - Added configuration for agent-query-cache, from @shamsimam
+### Changed
+- Replaced aggregate preemption logging with individual preemption decisions, from @wyegelwel
+- /debug endpoint now returns the version number, from @dposada
+### Experimental
+- Added support for Cook Executor, from @shamsimam
```
Comment (dposada) [inline CHANGELOG.md]: Why not just put this in "Added"?

**Comment by dposada** [comment]
@shamsimam I would favor no leading dashes in the commit messages :)

File: CHANGELOG.md (hunk 1 above)
Comment (shamsimam) [inline CHANGELOG.md]: Done.

File: CHANGELOG.md (hunk 3 above)
Comment (shamsimam) [inline CHANGELOG.md]: Done.

**Comment by shamsimam** [comment]
@dposada removed the leading dashes :)

**Comment by shamsimam** [comment]
Build is passing.

**Pull request merged**